tests/acceptance/features/apiContract/propfind.feature

```diff
@@ -120,3 +120,41 @@ Feature: Propfind test
       | Manager      | RDNVWZP       |
       | Space Editor | DNVW          |
       | Space Viewer |               |
+
+
+  Scenario: space-admin checks the audio, image, photo and location properties of PROPFIND request of file from project space
+    Given user "Alice" has uploaded a file "filesForUpload/testimage.mp3" to "testimage.mp3" in space "new-space"
+    And user "Alice" has uploaded a file "filesForUpload/testImage.jpg" to "testImage.jpg" in space "new-space"
+    When user "Alice" gets the following properties of resource "testimage.mp3" inside space "new-space" using the WebDAV API
+      | propertyName |
+      | oc:audio     |
+    Then the HTTP status code should be "207"
+    And the "PROPFIND" response should contain a space "new-space" with these key and value pairs:
+      | key                | value                          |
+      | oc:audio/oc:album  | ALBUM1234567890123456789012345 |
+      | oc:audio/oc:artist | ARTIST123456789012345678901234 |
+      | oc:audio/oc:genre  | Pop                            |
+      | oc:audio/oc:title  | TITLE1234567890123456789012345 |
+      | oc:audio/oc:track  | 1                              |
+      | oc:audio/oc:year   | 2001                           |
+    When user "Alice" gets the following properties of resource "testImage.jpg" inside space "new-space" using the WebDAV API
+        | propertyName |
+        | oc:image     |
+        | oc:location  |
+        | oc:photo     |
```

Comment:
testing these properties requires tika service. so maybe we can put this scenarios within `apiSearchContent` suite